app/main/views/evidence.py

```diff
@@ -30,6 +33,26 @@ def get_all_evidence_submitted():
     return jsonify(evidence=data), 200
 
 
+@main.route('/evidence/all', methods=['GET'])
+def get_all_evidence():
+    supplier_code = request.args.get('supplier_code', None)
+    evidence_collection = evidence_service.get_all_evidence(supplier_code=supplier_code)
+    data = []
+    for evidence in evidence_collection:
```

Comment:
Should we move the logic in these views to a file that sits under `/app/api/business`?

Reply:
The data these views return is rather specific for the part of the admin frontend that queries them - I'm not sure it will be useful for other parts. It's less logic and more building the object for the admin view that calls it.

Reply:
a repeat of @tobiasbrown, I think this should be in business and call business from here